__tests__/cache-utils.test.ts

```diff
@@ -174,6 +174,7 @@ describe('isCacheFeatureAvailable', () => {
       'Cache action is only supported on GHES version >= 3.5. If you are on version >=3.5 Please check with GHES admin if Actions cache service is enabled or not.';
 
     //Act + Assert
-    expect(() => cacheUtils.isCacheFeatureAvailable()).toThrow(errorMessage);
+    expect(cacheUtils.isCacheFeatureAvailable()).toBeFalsy();
+    expect(warningSpy).toHaveBeenCalledWith(errorMessage);
```

Comment:
Please rename the `errorMessage` variable to `warningMessage`. 
I think we also need to rename this test case to `should warn when cache feature is unavailable and GHES is used`

Reply:
@MaksimZhukov Thank you for your review. I pushed a commit.